projects/025 - Watkinson/template-sql/objectives-1-5.template.sql

```diff
@@ -114,6 +118,8 @@ SELECT
 	CASE WHEN HighVulnerabilityCodeDate IS NOT NULL THEN 'Y' ELSE 'N' END AS HasHighClinicalVulnerabilityIndicator,
 	HighVulnerabilityCodeDate AS DateOfHighClinicalVulnerabilityIndicator,
 	CASE WHEN mv.FK_Patient_Link_ID IS NOT NULL THEN 'Y' ELSE 'N' END AS HasModerateClinicalVulnerability,
+	CASE WHEN flu.FK_Patient_Link_ID IS NOT NULL THEN 'Y' ELSE 'N' END AS IsClinicallyEligibleForFluVaccine,
+	FluVaccineDate AS DateOfFluVaccineIn20192020Season,
```

Comment:
worth adding a prefix on FluVaccineDate like below to be explicit on where it's coming from
```suggestion
	fluvac.FluVaccineDate AS DateOfFluVaccineIn20192020Season,
```